examples/jupyter/tutorials/tutorial5-python-basis.ipynb

```diff
@@ -408,320 +412,4345 @@
-      " -3.20273120e-01 -3.08224298e-01 -2.82041576e-01 -2.34212849e-01\n",
-      " -1.57718860e-01 -5.61919791e-02  4.96805885e-02  1.35700996e-01\n",
-      "  1.91716972e-01  2.20128245e-01 -3.57425954e-01 -3.43979425e-01\n",
-      " -3.14759413e-01 -2.61382382e-01 -1.76014815e-01 -6.27104506e-02\n",
-      "  5.54437154e-02  1.51442800e-01  2.13956830e-01  2.45663912e-01\n",
-      "  4.96349757e-01  4.77676851e-01  4.37099646e-01  3.62976108e-01\n",
-      "  2.44427999e-01  8.70846580e-02 -7.69934986e-02 -2.10305368e-01\n",
-      " -2.97117261e-01 -3.41148207e-01  4.73554422e-01  4.55739087e-01\n",
-      "  4.17025429e-01  3.46306085e-01  2.33202410e-01  8.30852121e-02\n",
-      " -7.34574988e-02 -2.00646894e-01 -2.83471868e-01 -3.25480651e-01\n",
-      "  4.24559655e-01  4.08587525e-01  3.73879250e-01  3.10476653e-01\n",
-      "  2.09074882e-01  7.44890711e-02 -6.58574578e-02 -1.79887616e-01\n",
-      " -2.54143374e-01 -2.91805854e-01  3.36941718e-01  3.24265816e-01\n",
-      "  2.96720415e-01  2.46402445e-01  1.65927330e-01  5.91164877e-02\n",
-      " -5.22662122e-02 -1.42763548e-01 -2.01694872e-01 -2.31584808e-01\n",
-      "  2.01647725e-01  1.94061645e-01  1.77576695e-01  1.47463165e-01\n",
-      "  9.93016500e-02  3.53791311e-02 -3.12794830e-02 -8.54389438e-02\n",
-      " -1.20707261e-01 -1.38595333e-01  3.07118633e-02  2.95564689e-02\n",
-      "  2.70457362e-02  2.24593091e-02  1.51240918e-02  5.38840217e-03\n",
-      " -4.76400717e-03 -1.30127387e-02 -1.83842635e-02 -2.11086979e-02\n",
-      " -1.37834701e-01 -1.32649297e-01 -1.21381140e-01 -1.00797276e-01\n",
-      " -6.78768545e-02 -2.41831242e-02  2.13808423e-02  5.84011117e-02\n",
-      "  8.25084900e-02  9.47357394e-02 -2.68028168e-01 -2.57944825e-01\n",
-      " -2.36033192e-01 -1.96006585e-01 -1.31990774e-01 -4.70255923e-02\n",
-      "  4.15763804e-02  1.13564602e-01  1.60442901e-01  1.84219550e-01\n",
-      " -3.49764183e-01 -3.36605894e-01 -3.08012241e-01 -2.55779398e-01\n",
-      " -1.72241767e-01 -6.13661916e-02  5.42552258e-02  1.48196477e-01\n",
-      "  2.09370458e-01  2.40397869e-01 -3.90338086e-01 -3.75653387e-01\n",
-      " -3.43742768e-01 -2.85450727e-01 -1.92222432e-01 -6.84848904e-02\n",
-      "  6.05490270e-02  1.65387801e-01  2.33658184e-01  2.68284886e-01]\n"
+      "Xq = CeedVector length 10\n",
+      "  -0.973907\n",
+      "  -0.865063\n",
+      "  -0.679410\n",
+      "  -0.433395\n",
+      "  -0.148874\n",
+      "  0.148874\n",
+      "  0.433395\n",
+      "  0.679410\n",
+      "  0.865063\n",
+      "  0.973907\n",
+      "\n",
+      "Uq = CeedVector length 10\n",
+      "  -0.790923\n",
+      "  -0.746534\n",
+      "  -0.652361\n",
+      "  -0.487977\n",
+      "  -0.243859\n",
+      "  0.048837\n",
+      "  0.321566\n",
+      "  0.522238\n",
+      "  0.644049\n",
+      "  0.703354\n",
+      "\n",
+      "Xq = CeedVector length 200\n",
+      "  -0.973907\n",
+      "  -0.973907\n",
```

Comment:
Woah... this got huge

Reply:
I know right? I was hesitant too when I saw the git summary. Should I go back to the print array version for this example only?

Reply:
This is a function that 99% of users and 95% of developers will never call. I think this is another reason to put the advanced stuff in advanced tutorial notebooks.